examples/pipelines/datacomp/components/filter_text_complexity/src/main.py

```diff
@@ -27,6 +27,7 @@ def get_text_complexity(doc: spacy.tokens.doc.Doc):
     return complexity
 
 
+# this function can also be supported
 def get_num_actions(doc: spacy.tokens.doc.Doc):
```

Comment:
Would remove this function if it's not used. We have it in git, so can always re-add it when needed.